include/votca/tools/akimaspline.h

```diff
@@ -65,7 +63,7 @@ class AkimaSpline : public Spline {
 
   // fit spline through noisy data
   // x,y are arrays with noisy data, both vectors must be of same size
-  void Fit(Eigen::VectorXd &x, Eigen::VectorXd &y) override;
```

Comment:
I do not understand this change?

Reply:
x and y aren’t used in the function, so making them anonymous.